examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -240,21 +250,55 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 45,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Numerical data min,max,abs; -5.35e-03 +8.53e-03 +8.53e-03\n",
+      "Analytic  data min,max,abs; -5.32e-03 +8.54e-03 +8.54e-03\n"
+     ]
+    }
+   ],
    "source": [
-    "print(\"Numerical data min,max,abs; %+12.6e %+12.6e %+12.6e\" % \n",
+    "print(\"Numerical data min,max,abs; %+6.2e %+6.2e %+6.2e\" % \n",
     "      (np.min(ref_rec.data), np.max(ref_rec.data), np.max(np.abs(ref_rec.data)) ))\n",
-    "print(\"Analytic  data min,max,abs; %+12.6e %+12.6e %+12.6e\" % \n",
+    "print(\"Analytic  data min,max,abs; %+6.2e %+6.2e %+6.2e\" % \n",
```

Comment:
2 spaces, can remove one.

Reply:
the two spaces for "Analytic  " make it line up with "Numerical " above